OpenRA.Mods.RA2/OpenRA.Mods.RA2.csproj

```diff
@@ -63,6 +63,7 @@
         <Command type="AfterBuild" command="cp ${TargetFile}.mdb ../mods/ra2" workingdir="${ProjectDir}" />
       </CustomCommands>
     </CustomCommands>
+    <WarningLevel>4</WarningLevel>
```

Comment:
You end up better if you just revert this whole file and add the `Compile Include` line manually.